measures/ApplyUpgrade/measure.rb

```diff
@@ -47,6 +47,11 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     upgrade_name.setDefaultValue('My Upgrade')
     args << upgrade_name
 
+    project_directory = OpenStudio::Measure::OSArgument::makeStringArgument('project_directory', true)
+    project_directory.setDisplayName('Project Directory')
+    project_directory.setDescription('The directory containing the housing characteristics folder (e.g., project_national).')
+    args << project_directory
```

Comment:
Why is this new `project_directory` argument required, but the one in BuildExistingModel is optional?

Reply:
Both should be required. Let me change.